### PR TITLE
chore: add issue templates and readme

### PR DESCRIPTION
**Pull request by sfoster1**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/package_request.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+name: Feature request
+description: Request a new package
+title: 'package request: '
+labels: [package-request]
+body:
+  - type: input
+    attributes:
+      label: package link
+      description: A link to the package you'd like to have on the OT-2 on PyPI or condaforge or some other index
+      placeholder: https://pypi.org/project/pandas/
```
Comment (bndo) [inline .github/ISSUE_TEMPLATE/package_request.yml:11]: should we add "eg." or remove `pandas` from these?

**Review by bndo — COMMENTED** [review]
Tiny question

File: .github/ISSUE_TEMPLATE/package_request.yml (hunk 1 above)
Comment (sfoster1) [reply .github/ISSUE_TEMPLATE/package_request.yml:11]: This is how the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema) describe it:

> placeholder | A semi-opaque placeholder that renders in the text area when empty. | Optional | String | Empty String |  

Some of their examples have "ex." and some don't.

Comment (sfoster1) [reply .github/ISSUE_TEMPLATE/package_request.yml:11]: Let's try it without the `ex.` at first and we can change it later if it's not really as good as we think?

**Review by bndo — APPROVED** [review]

**Pull request merged**